tests/queries/0_stateless/03236_squashing_high_memory.sql

```diff
@@ -0,0 +1,22 @@
+DROP TABLE IF EXISTS id_values;
+
+DROP TABLE IF EXISTS test_table;
+
+CREATE TABLE id_values ENGINE MergeTree ORDER BY id1 AS
+    SELECT arrayJoin(range(1000000)) AS id1, arrayJoin(range(1000)) AS id2;
```

Comment:
```suggestion
    SELECT arrayJoin(range(500000)) AS id1, arrayJoin(range(1000)) AS id2;
```

is okay for me locally with 5GiB limit while the difference in memory usage is still high